qcodes/instrument/parameter.py

```diff
@@ -548,11 +580,15 @@ def get_ramp_values(self, value: Union[float, int, Sized],
 
     def validate(self, value: ParamDataType) -> None:
         """
-        Validate value
+        Validate the value supplied.
 
         Args:
-            value (any): value to validate
+            value: value to validate
 
+        Raises:
+            TypeError: If the value is of the wrong type.
+            ValueError: If the value is outside the bounds specified by the
+            validator.
```

Comment:
interesting that this line does not need extra indentation

Reply:
It probably does